client/src/component/layout/SearchableMaps.js

```diff
@@ -200,9 +199,32 @@ class SearchableMap extends Component {
       this.state.latlongSafety[i][2] = this.state.colors[i];
       this.state.latlongSafety[i][4] = this.state.sampleFeedback[i];
       this.state.latlongSafety[i][5] = i + 1;
+      this.state.latlongSafety[i][6] = {
```

Comment:
Hey @maruthimohan, the dimension of latlongSafety array is (no of routes)*6, so lalongSafety[i][6] will give out of bound error, so you should push the object in latlongSafety[i], like latlongSafety[i].push({object_you_created})

Reply:
Hey @iamdeepti ,  thank you for taking the time to review the PR. 

It is a valid property assignment in a JS Array as if the array index not in the bounds, it would add that as a new element. Below is the text quoted from the MDN documentation about that.

> When setting a property on a JavaScript array when the property is a valid array index and that index is outside the current bounds of the array, the engine will update the array's length property accordingly:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array

However, to avoid confusion, I have moved the distance and duration initialization to where each element for `latlongSafety` array is being created. And I have locally tested the App with the changes and it is working. 

Please let me know if that would work for you.